docker/arm-server/Dockerfile

```diff
@@ -1,30 +1,53 @@
-FROM ubuntu:22.04
-
+# Docker image for running CF instances in ARM64 server.
+# Docker image includes HO(Host Orchestrator) inside,
+# so it could execute CF instance with API in HO.
+FROM ubuntu:22.04 AS cuttlefish-arm64
```

Comment:
why do we need 'as cuttlefish-arm64' here?

Reply:
Just for setting target name & distinguishing for visibility. I don't have strong opinion on here.

Reply:
Close this conversation, since there looks no more discussion.